vtsite/vehicles/templates/vehicles/home.html

```diff
@@ -11,33 +11,35 @@
     <div class="form-container">
       {% if user_status.on_trip %}
         <!-- Display the ability to end trip -->
-        <h3>Your current trip:</h3>
-        <h4>Vehicle: <span>{{ trip.vehicle }}</span></h4>
-        <h4>Destination: <span>{{ trip.destination }}</span></h4>
-        <h4>Began trip at: <span>{{ trip.time_check_out }}</span></h4>
-        <button>Finish this trip</button>
+        <h3>Your current trip:</h3><br/>
+        <h4>Vehicle:  <span>{{ trip.vehicle }}</span></h4>
+        <h4>Destination:  <span>{{ trip.destination }}</span></h4>
+        <h4>Began trip at:  <span>{{ trip.time_check_out }}</span></h4><br/>
+        <a class="button-link" href="/vehicles/trip_finish"><button>Finish this trip</button></a>
```

Comment:
Please make this dynamic like {% url 'vehicles:trip_finish' %}. We can fix later 👍